enderio-conduits/src/main/java/com/enderio/conduits/common/network/ConduitServerPayloadHandler.java

```diff
@@ -85,4 +86,19 @@ public void handle(C2SOpenConduitFilterMenu packet, IPayloadContext context) {
             }
         });
     }
+
+    public void handle(C2SSyncProbeStatePacket packet, IPayloadContext context) {
+        context.enqueueWork(() -> {
```

Comment:
Maybe we should send the hand in the packet too? That way we definitely get the correct item when syncing.

Reply:
Actually, should we need to sync these details? The copying should really be performed on the server, not the client (some conduit options aren't synchronised to the client bundle anymore) - when it is updated on the server, it'll be synced automatically to the client.